### PR TITLE
UIU-1698: Create/Edit Patron Block: Move Save & close button to the f…

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
Create/Edit Patron Block: Move Save & close button to the footer pane.
Refs: [UIU-1698](https://issues.folio.org/browse/UIU-1698)
image
image

### CODE REVIEW

**Review by zburke — APPROVED** [review]
LGTM

File: src/components/PatronBlock/PatronBlockForm.js [hunk 1]
```diff
@@ -137,24 +138,44 @@ class PatronBlockForm extends React.Component {
   );
 
   renderLastMenu = () => {
+    const { params } = this.props;
+
+    const del = params.patronblockid ? <Button id="patron-block-delete" marginBottom0 buttonStyle="danger" onClick={this.props.onDeleteItem}><FormattedMessage id="ui-users.blocks.form.button.delete" /></Button> : '';
```
Comment (Dmitriy-Litvinenko) [inline src/components/PatronBlock/PatronBlockForm.js:143]: Could you please format this line?

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Review by zburke — APPROVED** [review]
LGTM

File: src/components/PatronBlock/PatronBlockForm.js (hunk 1 above)
Comment (UladzislauKutarkin) [reply src/components/PatronBlock/PatronBlockForm.js]: Done.

**Pull request merged**